src/djc_pydantic/__init__.py

```diff
@@ -0,0 +1,9 @@
+from djc_pydantic.monkeypatch import monkeypatch_pydantic_core_schema
```

Comment:
This file was still just renamed + updated imports, but git sees it as a new file